nixos/modules/tasks/network-interfaces.nix

```diff
@@ -565,6 +582,7 @@ in
             example = 100;
             type = types.nullOr types.int;
             description = ''
+              DEPRECATED, use `driverOptions`.
```

Comment:
Okay, after trying to reproduce this: This problem only applies to networkd. Investigating.